### PR TITLE
Dep notebooks

### PR DESCRIPTION
**Pull request by MichielStock**
Went through all the notebooks, minor changes:
- ⬆  dep Pluto to 1.6
- changed `Pkg; Pkg.install("DS-Julia")` to `Pkg; Pkg.activate("../..")` in the notebooks, using the local environment of our repo
- Deactivated some of the cells in the macro notebook (requiring JuMP and BioSequences). You can still show them but then they won't yield errors for students

Hope these work without trouble now!

### CODE REVIEW

File: notebooks/day1/01-basics-extra.jl [hunk 1]
```diff
@@ -5,7 +5,7 @@ using Markdown
 using InteractiveUtils
 
 # ╔═╡ 4109a39c-00b7-4954-bb73-35407f82a6f7
-using Pkg; Pkg.add(url="https://github.com/Beramos/DS-Julia2925")
+using Pkg; Pkg.activate("../..")
```
Comment (DaanVanHauwermeiren) [inline notebooks/day1/01-basics-extra.jl:8]: why is the DSJulia package environment activated here?
I thought we were going to keep these separated?
i.e. the env in the DSJulia only for the things in DSJulia and the env in notebooks for everything else you need in the course

**Review by DaanVanHauwermeiren — CHANGES_REQUESTED** [review]

**Comment by Beramos** [comment]
Agreed with Daan

Op zo 19 sep. 2021 17:30 schreef DaanVanHauwermeiren <
***@***.***>:

File: notebooks/day1/01-basics-extra.jl (hunk 1 above)
Comment (DaanVanHauwermeiren) [reply notebooks/day1/01-basics-extra.jl]: resolved in  4f01f53

**Comment by DaanVanHauwermeiren** [comment]
note: for metaprogramming in day2 I have added a project.toml file in notebooks/day2.
I thought that was the easiest solution

**Pull request merged**